docs/references-docs-in-vfs.md

```diff
@@ -113,16 +113,19 @@ Content-Type: application/vnd.api+json
 
 ```json
 {
-    "data": [
-        {
-            "type": "io.cozy.playlists",
-            "id": "94375086-e2e2-11e6-81b9-5bc0b9dd4aa4"
-        },
-        {
-            "type": "io.cozy.playlists",
-            "id": "f2625cc0-e2d6-11e6-a0d5-cfbbfb141af0"
-        }
-    ]
+    "data": {
+        "rev": "2-de8d0ba2",
```

Comment:
To be more consistent with JSON-API, the `rev` should be in `meta`, not in `data`:

```json
{
    "data": [
        {
            "type": "io.cozy.playlists",
            "id": "94375086-e2e2-11e6-81b9-5bc0b9dd4aa4"
        },
        {
            "type": "io.cozy.playlists",
            "id": "f2625cc0-e2d6-11e6-a0d5-cfbbfb141af0"
        }
    ],
    "meta": {
        "rev": "2-de8d0ba2"
    }
}
```